myst_nb/transform.py

```diff
@@ -49,3 +60,123 @@ def apply(self):
                 continue
             col = ImageCollector()
             col.process_doc(self.app, node)
+
+
+def cell_output_to_nodes_inline(cell, data_priority, write_stderr, dir, thebe_config):
```

Comment:
This one you'd like to upstream yeah? What's the main difference w/ jupyter-sphinx one?

Reply:
Yep. All the nodes returned are inline versions, i.e. `literal_block` -> `literal`, `container` -> `inline`,  `math_block` -> `math`